app/src/main/java/io/apicurio/registry/ccompat/rest/CompatibilityResource.java

```diff
@@ -39,7 +42,13 @@ public void testCompatabilityBySubjectName(
         @NotNull RegisterSchemaRequest request) throws Exception {
 
         // TODO - headers, level?
-        boolean isCompatible = rules.isCompatible(ArtifactType.avro, CompatibilityLevel.BACKWARD_TRANSITIVE, subject, request.getSchema());
+        boolean isCompatible = true;
+        try {
+            rules.applyRule(subject, ArtifactType.avro, request.getSchema(), RuleType.compatibility, 
+                    CompatibilityLevel.BACKWARD_TRANSITIVE.name(), RuleApplicationType.update);
```

Comment:
Can the enum value be passed here without the conversion to String?

Reply:
^ See my last comment, the `CompatibilityLevel` is converted to String which is again converted to `CompatibilityLevel`